packages/api/src/models/source.ts

```diff
@@ -14,7 +14,7 @@ export const Source = mongoose.model<ISource>(
     {
       kind: {
         type: String,
-        enum: ['log', 'trace'],
+        enum: ['log', 'trace', 'metric'],
```

Comment:
style: can we do `enum: Object.values(SourceKind)` ?

Reply:
Fixed